diff --git a/src/msw/notebook.cpp b/src/msw/notebook.cpp
index e82349f603..e368ec7689 100644
--- a/src/msw/notebook.cpp
+++ b/src/msw/notebook.cpp
@@ -1288,7 +1288,10 @@ void wxNotebook::MSWNotebookPaint(wxDC& dc)
         rectTabArea.SetRight(sizeWindow.x);
     else
         rectTabArea.SetBottom(sizeWindow.y);
-    dc.SetBrush(wxSystemSettings::GetColour(wxSYS_COLOUR_BTNFACE));
+    if (m_backgroundColour.IsOk())
+        dc.SetBrush(m_backgroundColour);
+    else
+        dc.SetBrush(wxSystemSettings::GetColour(wxSYS_COLOUR_BTNFACE));
     dc.SetPen(*wxTRANSPARENT_PEN);
     dc.DrawRectangle(rectTabArea);
 
